cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/KafkaCruiseControlConfig.java

```diff
@@ -438,6 +439,13 @@
       "movements the executor will take as one batch. This is mainly because the ZNode has a 1 MB size upper limit. And it " +
       "will also reduce the controller burden.";
 
+  /**
+   * <code>execution.task.strategy</code>
+   */
+  public static final String EXECUTION_TASK_STRATEGY_CONFIG = "execution.task.strategy";
```

Comment:
for `leadership_movement`, I feel that a customized strategy maybe too much since `leadership_movement` is a resource-cheap operation. And it is unrelated with whether partition is large or not, or is partition under replicated(since in `replica_movement` stage all the URP are recovered).

Reply:
But unless the unclean leader election is enabled, wouldn't the leadership transfer take `URPs` into account?

Reply:
That is a good point. After a second thought, I feel in that case we may still not need to postpone leadership change operation for `URP` because when Kafka is handling leadership change, it does not wait URP to recovery, seems it makes no difference either partition is under replicated or not.